src/components/ChangePlan/PlanCalculator.js

```diff
@@ -157,16 +157,20 @@ const PlanPriceWithoutDiscounts = ({ planData }) => {
 const PlanPricePerMonth = ({ planData }) => {
   const intl = useIntl();
   const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
   return (
     <>
       <h2 className="dp-price-large">
         <span className="dp-price-large-money">US$</span>
         <span className="dp-price-large-amount">
-          {planData.discount?.discountPercentage
-            ? Math.round(
-                getPlanFee(planData.plan) * (1 - planData.discount?.discountPercentage / 100),
-              )
-            : getPlanFee(planData.plan)}
+          {thousandSeparatorNumber(
+            intl.defaultLocale,
+            planData.discount?.discountPercentage
+              ? Math.round(
+                  getPlanFee(planData.plan) * (1 - planData.discount?.discountPercentage / 100),
+                )
+              : getPlanFee(planData.plan),
+          )}
```

Comment:
it's not necessary planData.discount?.discountPercentage again, because was already did it. 
planData.discount.discountPercentage working fine.

You could to use a constant to save the planData.discount?.discountPercentage value, as it is used on several sides. It's more simple and you use DRY. 
Maybe: 

const discountPercentage = planData.discount?.discountPercentage;

{discountPercentage
            ? Math.round(
                getPlanFee(planData.plan) * (1 - discountPercentage / 100),  etc

Reply:
Thanks for the suggestion! I have already merge this PR, but I have to apply some changes to resolve an issue with prices rounding, so I will take in count this comment!